clients/venice-client/src/main/java/com/linkedin/venice/fastclient/DispatchingAvroGenericStoreClient.java

```diff
@@ -467,6 +481,10 @@ private void transportRequestCompletionHandler(
     if (exception != null) {
```

Comment:
This used to only skip the transport client for one route and set the partial response exception flag. The overall request will have an optional exception set in the `whenComplete` callback. 
```
callback.onCompletion(Optional.of(new VeniceClientException("At least one route did not complete", clientException)));
```
Based on this and `BatchGetRequestContext`'s implementation I don't think one route will slowdown the entire request but we can sync up offline to resolve the confusion.